xml/System.Diagnostics.Tracing/EventSource.xml

```diff
@@ -3820,7 +3820,7 @@ desc.DataPointer = (IntPtr)(&arg);
 desc.Size = UIntPtr.Size;
 
 // DateTime arg
-long fileTime = arg.ToFileTimeUtc();
+long fileTime = arg.Ticks > 504_911_232_000_000_000 ? arg.ToFileTimeUtc() : 0;
```

Comment:
> arg.Ticks > 504_911_232_000_000_000  [](http://example.com/codeflow?start=15&length=37)

Would be better to write this as 

```C$
arg.ToUniversalTime() > new DateTime(1601, 1, 1)
```

This is more accurate to compare the Utc ticks of the time and also better for readability.

Reply:
Good catch, thanks. Will update